app/jobs/state_file/after_transition_tasks_for_rejected_return_job.rb

```diff
@@ -9,9 +9,10 @@ def perform(submission, transition)
         if transition.efile_errors.any?(&:auto_cancel)
           submission.transition_to!(:notified_of_rejection)
           submission.transition_to!(:cancelled)
+        elsif transition.efile_errors.all?(&:auto_wait)
+          submission.transition_to!(:notified_of_rejection)
```

Comment:
in the `else` scenario, they remain in `rejected` state & see `pending` html on `return-status` and do not receive any notification.